### PR TITLE
addPhone

### PR DESCRIPTION
**Pull request by lxc150896**
task: https://dev.framgia.com/issues/211520

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -101,4 +106,26 @@ public function search(SearchRequest $request)
...
+            $id = Auth::id();
+            $data = [
+                'phone' => $phone,
+            ];
+            $this->user->update($id, $data);
+
+            return response()->json([
+                'data' => '1',
+            ]);
+        } else {
```
Comment (framgiaciv3) [inline app/Http/Controllers/HomeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 124</br>Blank line found at start of control structure

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -101,4 +106,26 @@ public function search(SearchRequest $request)
...
+            $id = Auth::id();
+            $data = [
+                'phone' => $phone,
+            ];
+            $this->user->update($id, $data);
+
+            return response()->json([
+                'data' => '1',
+            ]);
+        }else {
```
Comment (framgiaciv3) [inline app/Http/Controllers/HomeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 124</br>Expected 1 space after closing brace; 0 found

Comment (framgiaciv3) [inline app/Http/Controllers/HomeController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 124</br>Blank line found at start of control structure

File: resources/assets/user/js/main.js [hunk 3]
```diff
@@ -671,6 +671,30 @@
...
+        $('#myModal').modal('show');
+    });
+
+    $('#modal_phone').on('click', function() {
+        $.ajax({
+            url: '/my-phone/' + $('#phone_value').val(),
+            method:'POST',
+            success: function(res) {
+                if (res.data == 1) {
+                    swal('Good job!', 'Success!', 'success');
```
Comment (danhloimta) [inline resources/assets/user/js/main.js]: i18n

File: resources/assets/user/js/main.js [hunk 4]
```diff
@@ -671,6 +671,30 @@
...
+                    $('#myModal').modal('toggle');
+                } else {
+                    swal({
+                      title: 'You enter a non-phone number!',
+                      text: 'Select ok to re-enter!',
+                      icon: 'warning',
+                      dangerMode: true,
```
Comment (danhloimta) [inline resources/assets/user/js/main.js:699]: tương tự

File: resources/assets/user/js/main.js [hunk 5]
```diff
@@ -671,6 +671,30 @@
...
     });
 
+    $(window).on('load', function(event) {
+        $('#myModal').modal('show');
+    });
+
+    $('#modal_phone').on('click', function() {
+        $.ajax({
+            url: '/my-phone/' + $('#phone_value').val(),
+            method:'POST',
```
Comment (danhloimta) [inline resources/assets/user/js/main.js]: convention

**Pull request merged**